Source/WebKit/UIProcess/WebFrameProxy.cpp

```diff
@@ -510,11 +510,14 @@ void WebFrameProxy::didCreateSubframe(WebCore::FrameIdentifier frameID, String&&
     child->m_parentFrame = *this;
     child->m_frameName = WTFMove(frameName);
     page->observeAndCreateRemoteSubframesInOtherProcesses(child, child->m_frameName);
+#if ENABLE(WEBDRIVER_BIDI)
+    Ref childPtr = child.get();
```

Comment:
This is weird. It would be preferable to change WTFMove(child) into child.copyRef(). That will extend the lifetime of `child` long enough.